sdk/python/kfp/_client.py

```diff
@@ -710,12 +710,8 @@ def run_pipeline(
           version_id: The id of a pipeline version.
             If both pipeline_id and version_id are specified, version_id will take precendence.
             If only pipeline_id is specified, the default version of this pipeline is used to create the run.
-          pipeline_root: The root path of the pipeline outputs. This argument should
```

Comment:
Keep the docstring since you are not deleting the argument from function signature?